api-reference/_hidden/GridBaseColumn/isBand.md

```diff
@@ -5,54 +5,16 @@ default: undefined
 ---
 ---
 ##### shortDescription
-Specifies whether the column bands other columns or not.
+Specifies whether the column bands other columns.
 
 ---
-Unlike normal columns, band columns do not hold data. Instead, they collect two or more columns under one column header. In most cases, to set up this layout, you can declare the band column using a hierarchical structure. For example, the following code bands three columns under the *"Address"* header.
+Unlike normal columns, band columns do not hold data. Instead, they collect two or more columns under one column header. To create such a layout, do one of the following:
```

Comment:
`Instead, they collect two or more columns under one column header.`

Can you clarify what you mean here?

Reply:
![image](https://user-images.githubusercontent.com/66619750/91981685-e0dd4780-ed31-11ea-83f8-09b90cd8baa8.png)
In this picture, the `Nominal GDP` column collects the `Total, mln` and `By Sector` columns under its header.

Reply:
Thanks. Can we say the following?

```suggestion
Unlike normal columns, banded columns do not contain data. Instead, a banded column is a column header that displays two or more columns underneath it. To create a banded layout, do one of the following:
```

Reply:
No, because banded columns contain data (e.g.: Agriculture, Inductry, Services columns from my previous comment). 
It's better to change "Instead, a banded column is a column header that displays two or more columns underneath it." to "Instead, a band column  collects two or more columns under its header." because a band column is a column, not a header.
Thus the paragraph is as follows:
'''
Unlike standard columns, band columns do not contain data. Instead, a band column  collects two or more columns under its header. To create a banded layout, do one of the following:
'''
Is it good?

Reply:
I have a minor change request. I was not able to find instances of "band columns" in Microsoft's documentation, so would it be okay to change this to "banded columns"? For example:

```suggestion
Unlike standard columns, banded columns do not contain data. Instead, a banded column collects two or more columns under its header. To create a banded layout, do one of the following:
```

Reply:
Nevertheless, in DevExtreme, band (not banded) column is used: https://docs.devexpress.com/WPF/117796/controls-and-libraries/data-grid/visual-elements/common-elements/band-column
That's why the paragraph should be 
"""
Unlike standard columns, band columns do not contain data. Instead, a band column collects two or more columns under its header. To create a banded layout, do one of the following:
"""

Is it alright?

Reply:
Okay, I understand.